tox.ini

```diff
@@ -735,6 +736,9 @@ commands =
 
     sleep 30
 
+    # Patches CoreDNS
+    tox -e dev.k8s.patch.coredns
```

Comment:
We can call patch before the `sleep 30` and right after cluster testdomain. This way the config is available during before the deploy where it will be used. Else coredns will only refresh every 15s